environment.yml

```
@@ -2,24 +2,5 @@ name: talkpython-dask
 channels:
   - conda-forge
 dependencies:
-  - python=3.8
-  - nodejs
-  - dask>=2021.3.0
-  - dask-ml>=1.7.0
-  - distributed>=2021.3.0
-  - jupyterlab>=3.0
-  - notebook
-  - pandas>=1.0.1
-  - numpy>=1.19.2
-  - scipy>=1.4.1
-  - scikit-learn>=0.22.1
-  - scikit-image>=0.15.0
-  - ipywidgets>=7.5
-  - bokeh>=2.3.0
-  - pip>=20.3.0
-  - pip:
-    - dask-labextension>=3.0.0
-    - coiled
-  - python-graphviz
-  - h5py
-  - mimesis
+  - python=3.9
+  - coiled-runtime>=0.2.0
```

let's do >= 0.2.1 (since this one has the latest pyarrow)
```suggestion
  - coiled-runtime>=0.2.1
```